setup.py

```diff
@@ -1,77 +1,39 @@
-#!/usr/bin/env python
+#! /usr/bin/env python
 # -*- coding: utf-8 -*-
 '''
 The setup script for SaltPyLint
 '''
 # pylint: disable=file-perms,wrong-import-position
 from __future__ import absolute_import, with_statement
-import io
 import os
-import sys
 
-SETUP_KWARGS = {}
-USE_SETUPTOOLS = False
 
-# Change to salt source's directory prior to running any command
-try:
-    SETUP_DIRNAME = os.path.dirname(__file__)
-except NameError:
-    # We're most likely being frozen and __file__ triggered this NameError
-    # Let's work around that
-    SETUP_DIRNAME = os.path.dirname(sys.argv[0])
-
-
-if SETUP_DIRNAME != '':
-    os.chdir(SETUP_DIRNAME)
-
-SALT_PYLINT_REQS = os.path.join(os.path.abspath(SETUP_DIRNAME), 'requirements.txt')
+BASE_DIR = os.path.dirname(__file__)
 
+INSTALL_REQUIRES = [
```

Comment:
I get not wanting to maintain two sources so why have the requirements file at all?

Based on [recommended packaging conventions](https://packaging.python.org/discussions/install-requires-vs-requirements/) it seems `install_requires` is the right place to define these abstract requirements rather than maintaining a file and parsing it in a non-standard way.